notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -80,9 +80,9 @@
        "      <th>sex</th>\n",
```

Comment:
<p>Hey this is a <code>test</code> ?</p><p> </p><p>Are you ready ?</p><p> </p><p> </p><p> </p><p>after this multi line break</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>